QuickDeployment/QuickDeploymentInternetSetup.md

```diff
@@ -0,0 +1,215 @@
+
+### SSL Certificate Scenarios
+
+There are three possible approaches to acquiring SSL certificates for QDE:
+
+* [Scenario 1: Domain Server Certificates](#scenario-1-domain-server-certificates): This would be the domain server SSL certificate, generated on domain join and trusted by all endpoints on your domain.
+* [Scenario 2: Purchased/Acquired Certificates from CA](#scenario-2-purchasedacquired-certificates-from-ca): This would be an SSL certificate that you purchased or acquired from an external Certificate Authority (CA; e.g. LetsEncrypt).
+* [Scenario 3: Self-Signed SSL Certificates](#scenario-3-self-signed-ssl-certificates): This would be a self-generated SSL certificate, similar to the one that comes with QDE (least-desirable).
+
+Each approach is discussed individually in the sections below.
+
+### Scenario 1: Domain Server Certificates
+
+If you manage a Windows AD Domain, this is going to be the most common scenario. The majority of customers will want to join the QDE server to their domain, and possibly even change the hostname to match their organizational naming conventions. As this process will change your FQDN, please go ahead and do this **_first_**, before proceeding any further in this document.
+
+Once you have your new valid FQDN associated to your QDE server (that is DNS-resolvable on all your endpoints), you will now want to ensure the domain server certificate associated with that FQDN is utilized for the Nexus and CCM services on QDE.
+
+Firstly, open the "Certificates - Local Computer" MMC snap-in by pressing the Windows key, and when the Start menu pops up, type certificates. You should now see an option under the "Settings" section that says "Manage computer certificates". Alternatively, open the Run dialog (Windows key + R), type `certlm.msc`, and click "OK".
+
+Under the "Personal" store, you should see a server certificate matching the FQDN of your QDE server. Double-click on the certificate to open it, and under the details tab, copy out the `Thumbprint` value of the certificate. You will require this for later steps.
+
+Now, you can run the `New-SslCertificates.ps1` script and pass the thumbprint you copied above using the `-Thumbprint` parameter. Please run the below command from a PowerShell Administrator window:
+
+```powershell
+Set-ExecutionPolicy Bypass -Scope Process -Force; . C:\choco-setup\files\New-SslCertificates.ps1 -Thumbprint '<YOUR_CERT_THUMBPRINT_HERE>'
+```
+
+In this Domain scenario, it is assumed that the endpoints connecting to Nexus and CCM are also on the same domain. As such, they will inherently trust the domain server certificate of the QDE server. If some endpoints are not on the domain, or on a different domain, you will need to ensure that the Nexus and CCM certificates of QDE are copied to the `Local Computer\Trusted People\Certificates` store on those endpoints as well.
```

Comment:
While this is really good information, please make sure there is an ordered list that lays out exactly what needs to be done. I see it in the information here, but not everyone is going to be able to draw out the exact steps they need to do. That said, it is good to elaborate on them as well (as you have already done here).

This applies to all 3 scenarios.

Reply:
Good idea Rob. I've reworked these sections into ordered lists, for clarity.